skimage/util/tests/test_arraypad.py

```diff
@@ -86,11 +153,11 @@ def test_check_minimum_1(self):
         a = np.arange(100)
         a = pad(a, (25, 20), 'minimum')
         b = np.array(
-            [0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
-             0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
-             0,  0,  0,  0,  0,
+            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
Here too. There is *nothing* different about the second-to-last zero in terms of character code.